vividus-to-xray-for-jira-exporter/src/main/java/org/vividus/xray/exporter/exception/XrayExporterException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.xray.exporter.exception;
+
+public class XrayExporterException extends RuntimeException
```

Comment:
why `Runtime`?

Reply:
Currently this exception used in the converter and throws on wrong formatted data provided by used, I think it's worth to rename it to make more accurate